doc/GETTING_STARTED.md

```diff
@@ -0,0 +1,428 @@
+- In the event of working on a browser runtime, you must go with `js-libp2p-webrtc-star`.
+- if you need to crawl the network and discover a large number of peers, `js-libp2p-kad-dht`.
+
+Reviewing the events specification on [Libp2p.create](./API.md#create), each time a peer is discovered a `peer:discovery` event will be emitted by the node.
+
+We will consider in this guide that we already know some peers and also that we are creating an application that might be used by people within the same office network. Accordingly, we will install `libp2p-bootstrap` and `libp2p-mdns`
+
+```sh
+npm install libp2p-bootstrap libp2p-mdns
+```
+
+We can provide specific configurations for each protocol within a `config.peerDiscovery` property in the options.
+
+```js
+const Libp2p = require('libp2p')
+const TCP = require('libp2p-tcp')
+const SECIO = require('libp2p-secio')
+const MPLEX = require('libp2p-mplex')
+
+const Bootstrap = require('libp2p-bootstap')
+const MulticastDNS = require('libp2p-mdns')
+
+// Known peers addresses
+const bootstrappers = [
+  '/ip4/104.131.131.82/tcp/4001/p2p/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ',
+  '/ip4/104.236.176.52/tcp/4001/p2p/QmSoLnSGccFuZQJzRadHn95W2CrSFmZuTdDWP8HXaHca9z'
+]
+
+const node = await Libp2p.create({
+  modules: {
+    transport: [TCP],
+    connEncryption: [SECIO],
+    streamMuxer: [MPLEX],
+    peerDiscovery: [Bootstrap, MulticastDNS]
+  },
+  config: {
+    peerDiscovery: {
+      autoDial: true, // Auto connect to discovered peers (limited by ConnectionManager minPeers)
+      // The `tag` property will be searched when creating the instance of your Peer Discovery service.
+      // The associated object, will be passed to the service when it is instantiated.
+      [MulticastDNS.tag]: {
+        interval: 1000,
+        enabled: true
+      },
+      [Bootstrap.tag]: {
+        interval: 20000,
+        enabled: true,
+        list: bootstrapers // provide array of peers
+      }
+    }
+  }
+})
+
+node.on('peer:discovery', (peer) => {
+  console.log(peer) // Log discovered peer
+})
+
+// start libp2p
+await node.start()
+```
+
+<details><summary>Read More</summary>
+If you want to know more about libp2p peer discovery, you should read the following content:
+
+- https://github.com/libp2p/specs/blob/master/discovery/mdns.md
+</details>
+
+#### Peer Routing
+
+Peer Routing offers a way to find other peers in the network by issuing queries using a Peer Routing algorithm, through their peer ids.
+
+Looking at the [available peer routing](./CONFIGURATION.md#peer-routing) protocols, we might choose `libp2p-kad-dht` or `libp2p-delegated-peer-routing`. If your runtime is a browser or a low power device, you might prefer to use `libp2p-delegated-peer-routing` as you are able to delegate other peer to issue the queries for you. However, you need to know a peer able to do that. Otherwise, you should use `libp2p-kad-dht`.
```

Comment:
```suggestion
Looking at the [available peer routing](./CONFIGURATION.md#peer-routing) modules, we might choose `libp2p-kad-dht` or `libp2p-delegated-peer-routing`. If your runtime is a browser or a low power device, you might prefer to use `libp2p-delegated-peer-routing` as you are able to delegate to another peer to issue the queries for you. However, you need to know a peer able to do that. Otherwise, you should use `libp2p-kad-dht`.
```